### ISSUE TITLE
Error with --HMR option in Nativescript-Vue 2.2.0 

### ISSUE THREAD

**Issue by hoatructhanhlam** [nativescript-vue/nativescript-vue#461]
### Version
2.2.0

### Reproduction link
[https://github.com/hoatructhanhlam/bug-hmr-nativescript-vue](https://github.com/hoatructhanhlam/bug-hmr-nativescript-vue)
### Platform and OS info
Android 7.0

### Steps to reproduce
tns run android --hmr

### What is expected?
After the launch, the page transfer application is normal with
`this.$navigateTo(Template)`
But after app rerender with --hmr option, if redirect to new page, will get exception

### What is actually happening?
```
System.err: java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.
System.err: 	at android.view.ViewGroup.addViewInner(ViewGroup.java:5161)
System.err: 	at android.view.ViewGroup.addView(ViewGroup.java:4991)
System.err: 	at android.view.ViewGroup.addView(ViewGroup.java:4931)
System.err: 	at android.view.ViewGroup.addView(ViewGroup.java:4904)
System.err: 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1427)
System.err: 	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1752)
System.err: 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1821)
System.err: 	at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
System.err: 	at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2595)
System.err: 	at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2382)
System.err: 	at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2337)
System.err: 	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2244)
System.err: 	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:702)
System.err: 	at android.os.Handler.handleCallback(Handler.java:790)
System.err: 	at android.os.Handler.dispatchMessage(Handler.java:99)
System.err: 	at android.os.Looper.loop(Looper.java:164)
System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7000)
System.err: 	at java.lang.reflect.Method.invoke(Native Method)
System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:441)
System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
```

P/S: Before that (v2.0.2), it still works great

**Comment by hoatructhanhlam** [comment]
I have tested on multiple versions (even downgraded platforms) also cannot solve the problem. My development experience is significantly affected.
I wonder if anyone has a situation like mine?

**Comment by tralves** [comment]
The **Reproduction link** is not correct. Do you have some code sample where this happens?

**Comment by hoatructhanhlam** [comment]
> The **Reproduction link** is not correct. Do you have some code sample where this happens?

Sorry! I justed updated it.
Run `tns run android --hmr`
then change any code and page redirection will see the error

**Comment by msaelices** [comment]
@hoatructhanhlam The issue was fixed here: https://github.com/nativescript-vue/nativescript-vue/commit/595d2a0032de2b7b45805ef0a5266e6fd4dbe820

**Comment by msaelices** [comment]
@hoatructhanhlam until a new version was released you can check it by doing this:

1. Clone this repo. Run `npm install && npm run build` inside it.
2. Change the `nativescript-vue` dependency in your app's `package.json` to `"nativescript-vue": "file://path/to/nativescript-vue"`
3. In your app, remove your current `node_modules/nativescript-vue` and run `npm install` again to make a link to the git version.
4. Run `tns run android --hmr`

**Comment by hoatructhanhlam** [comment]
@msaelices thank you! that's a temporary solution for me

**Comment by msaelices** [comment]
@hoatructhanhlam good news! @rigor789 just released a new 2.2.1 version which includes my fix. I've just tested it with your repo (changing your  `package.json` dependency to `^2.2.1`) and it works.

I will close the issue. Please re-open if there is something I missed.
